custom/icds/views.py

```diff
@@ -0,0 +1,111 @@
+from django import forms
+from django.core.urlresolvers import reverse
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext as _, ugettext_lazy
+
+from crispy_forms import bootstrap as twbscrispy
+from crispy_forms import layout as crispy
+from crispy_forms.helper import FormHelper
+
+from corehq.apps.domain.decorators import require_superuser_or_developer
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.es.users import UserES
+from corehq.apps.style import crispy as hqcrispy
+from corehq.apps.style.decorators import use_select2
+
+from custom.icds.messaging.indicators import (
+    AWWSubmissionPerformanceIndicator,
+    LSSubmissionPerformanceIndicator,
+    LSVHNDSurveyIndicator,
+)
+from custom.icds.tasks import run_indicator
+
+
+class IndicatorTestForm(forms.Form):
```

Comment:
normally i think we'd want all forms to go in their own file, but i won't block on this, we're going to remove this when testing is done anyway